src/main/java/stringpluscalculator/Separators.java

```diff
@@ -0,0 +1,55 @@
+package stringpluscalculator;
+
+import java.util.Arrays;
+
+public class Separators {
+    private static final String CUSTOM_SEPARATORS_PREFIX = "//";
+    private static final String CUSTOM_SEPARATORS_SUFFIX = "\\n";
+    private static final String SPLIT_DELIMITER = "|";
+
+    private String[] separators = {":", ","};
+
+    public Separators(String input) {
+        if (hasCustomSeparator(input)) {
+            this.separators = new String[]{searchCustomSeparator(input)};
+        }
+    }
+
+    public String getSplitSeparator() {
+        return String.join(SPLIT_DELIMITER, this.separators);
+    }
+
+    public static String getInputWithoutCustomSeparator(String input) {
+        if (hasCustomSeparator(input)) {
+            return input.substring(input.indexOf(CUSTOM_SEPARATORS_SUFFIX) + CUSTOM_SEPARATORS_SUFFIX.length());
+        }
+        return input;
+    }
+
+    private static boolean hasCustomSeparator(String input) {
+        if (input.startsWith(CUSTOM_SEPARATORS_PREFIX) &&
+                input.indexOf(CUSTOM_SEPARATORS_SUFFIX) > CUSTOM_SEPARATORS_PREFIX.length()) {
+            return true;
+        }
+        return false;
```

Comment:
아래의 코드처럼 한 줄로 줄일 수 있어요.

```suggestion
        return input.startsWith(CUSTOM_SEPARATORS_PREFIX) &&
                input.indexOf(CUSTOM_SEPARATORS_SUFFIX) > CUSTOM_SEPARATORS_PREFIX.length();
```